src/tests/integration/test_foundry_prove.py

```diff
@@ -410,7 +410,7 @@ def test_foundry_remove_node(
     foundry_remove_node(
         foundry,
         test=test,
-        node=4,
+        node=8,
```

Comment:
This needed to be changed because the minimization of the KCFG of the associated `setUp` function resulted in a change of node numbering.